### ISSUE TITLE
the loss of the finetune

### ISSUE THREAD

**Issue by Dailleson** [Ugness/PiCANet-Implementation#16]
Thanks for your implementation.when I finetune the model using your train.py  with the pre-trained model on my own data ,the loss didn't convergence.Do you have any suggesstion?

**Comment by zicair** [comment]
> Thanks for your implementation.when I finetune the model using your train.py with the pre-trained model on my own data ,the loss didn't convergence.Do you have any suggesstion?

I had the same problem，loss can only converge to about 1.0---1.5, It's no longer going down. is that the case for you？？

**Comment by Ugness** [comment]
![image](https://user-images.githubusercontent.com/33034982/55485624-c3500e00-5665-11e9-8218-734f5282d2ce.png)
This is my experiment's result. 
Since I reset my computer, I need to set the environment again to train it again. Please wait for that.
Since the loss is not the average value over the whole epoch and it is the loss of each batch, it may seem like diverging or vibrating. But it is still in training. So I suggest you to check
1. The actual output image
2. Measure the MAE / f-measure score by measure_test.py
3. Plot the loss-epoch graph or use the smooth option in tensorboard. It may show a declining trend.

**Comment by zicair** [comment]
I have another question，the loss in your code is  0.5 0.5..0.8..1，But here's the author's code
![image](https://user-images.githubusercontent.com/41500153/55491535-378bb100-5668-11e9-8b79-33a66351d6f9.png)
I would like to know why it is not the same as the one set by the author. Looking forward to your reply, thank you very much！

**Comment by Ugness** [comment]
@zicair
In the PiCANet paper Chapter 5.3, 
The whole net-
work is trained end-to-end using stochastic gradient descent
(SGD) with momentum. Since deep supervision is adopted
in each decoding module, we empirically weight the losses
in D7
, D5
, D4
, · · · , D1 by 0.5, 0.5, 0.5, 0.8, 0.8, and 1,
respectively, without further tuning. 
And I followed the coefficients in the paper

**Comment by RaoHaobo** [comment]
@Ugness  ,I change my learning_rate,this is my best train loss,
![image](https://user-images.githubusercontent.com/44016031/55779764-edc42000-5ad8-11e9-9838-f2c325159f67.png)
but I test it on PASCAL-S ,the max F-score  is 0.82-0.83,I think it maybe poor.
